framework/base/src/types/interaction/tx_call_te.rs

```diff
@@ -49,4 +49,12 @@ where
     pub fn transfer(self) {
         self.transfer_execute_with_gas(0)
     }
+
+    pub fn transfer_non_zero(self) {
```

Comment:
Add some docs with what it does "Transfers funds, amount is greater than zero. Does nothing otherwise"